front/src/app/index.tsx

```diff
@@ -1,22 +1,42 @@
 import React from "react";
-import { ContainerCustom } from "./app.styled";
+import { CardsContainer, CardElement } from "./app.styled";
 
-export const App = () => {
-  const [count, setCount] = React.useState(0);
+function Card(props: any){
+  return (
+    <CardElement>
+      <div className="card-upper-section">
+        <div className="card-image-container"> </div>
+        <p>{props.response.status}</p>
+      </div>
+      <div className="card-lower-section">
+        <p>{props.response.name}</p>
+      </div>
```

Comment:
esses elementos aqui poderia ser direto aqueles styled que comentei uns comentarios acima. 

```suggestion
      <CardUpperSection>
        <CardImageContainer />
        <CardTextStyle>{props.response.status}</CardTextStyle>
      </CardUpperSection>
      <CardLowerSection>
        <CardTextStyle>{props.response.name}</CardTextStyle>
      </CardLowerSection>
```

só teria que criar alem dos que comentei o `CardTextStyle` por exemplo, para substituir o p. Se não quiser dar style nenhum para o componente, da pra colocar a string vazia. Exemplo:

```typescript
export const CardTextStyle = styled.p``;
```